front/app/translations/de-DE.json

```diff
@@ -562,21 +562,21 @@
   "app.containers.AccessibilityStatement.contentConformanceExceptions": "Wir bemühen uns unsere Inhalte für alle inklusiv zu gestalten. Allerdings sind bestimmte Inhalte der Plattform unter den folgenden Umständen nicht barrierefrei:",
   "app.containers.AccessibilityStatement.demoPlatformLinkText": "Demo-Webseite",
   "app.containers.AccessibilityStatement.email": "E-Mail:",
-  "app.containers.AccessibilityStatement.exception_1": "Unsere Beteiligungsplattformen fördern die Veröffentlichung von nutzergenerierten Inhalten von Einzelpersonen und Organisationen. Sie ermöglichen den Nutzer:innen unserer Plattformen das Hochladen von PDFs, Bildern, Multimediadateien und anderen Dateitypen in Form von Anhängen oder als Einfügungen in die Textfelder. Möglicherweise sind diese Dokumente nicht vollständig zugänglich.",
+  "app.containers.AccessibilityStatement.exception_1": "Unsere Beteiligungsplattformen fördern die Veröffentlichung von nutzergenerierten Inhalten von Einzelpersonen und Organisationen. Sie ermöglichen den Nutzern unserer Plattformen das Hochladen von PDFs, Bildern, Multimediadateien und anderen Dateitypen in Form von Anhängen oder als Einfügungen in die Textfelder. Möglicherweise sind diese Dokumente nicht vollständig zugänglich.",
   "app.containers.AccessibilityStatement.feedbackProcessIntro": "Wir freuen uns über Rückmeldungen zur Barrierefreiheit dieser Website. Bitte kontaktieren Sie uns auf einem der folgenden Wege:",
   "app.containers.AccessibilityStatement.feedbackProcessTitle": "Feedbackprozess",
-  "app.containers.AccessibilityStatement.intro2022": "{citizenLabLink} ist bestrebt, eine Plattform bereitzustellen, die für alle Nutzer:innen zugänglich ist, unabhängig von Technologie oder Fähigkeiten. Bei unseren ständigen Bemühungen, die Zugänglichkeit und Nutzbarkeit unserer Plattformen für alle Nutzer:innen zu maximieren, halten wir uns an die aktuellen einschlägigen Standards zur Barrierefreiheit.",
+  "app.containers.AccessibilityStatement.intro2022": "{citizenLabLink} ist bestrebt, eine Plattform bereitzustellen, die für alle Nutzer zugänglich ist, unabhängig von Technologie oder Fähigkeiten. Bei unseren ständigen Bemühungen, die Zugänglichkeit und Nutzbarkeit unserer Plattformen für alle Nutzer zu maximieren, halten wir uns an die aktuellen einschlägigen Standards zur Barrierefreiheit.",
```

Comment:
I'm not sure the changes to this file that remove the uses of `:` in copy are correct. I remember Jelena telling me that this is used to indicate a gender-neutral phrase.

@lilkraftwerk What would be a good way to raise this issue? Should I go ahead and merge this PR, or some other course of action?

Reply:
I confirmed with Jelena that these changes are intentional. [Slack thread](https://citizenlabco.slack.com/archives/C03L5NGFY79/p1658928893394449).